rest/config.go

```diff
@@ -45,8 +45,8 @@ var (
 	// The value of defaultLogFilePath is populated by --defaultLogFilePath in ParseCommandLine()
 	defaultLogFilePath string
 
-	// ErrUnknownField is marked as the cause of the error when trying to decode a JSON config with unknown fields
-	ErrUnknownField = errors.New("unrecognized config value")
+	// ErrUnknownField is marked as the cause of the error when trying to decode a JSON snippet with unknown fields
+	ErrUnknownField = errors.New("unrecognized JSON field")
```

Comment:
We've got this defined in two places now - this one is unused. I think you've accidentally reintroduced it instead of changing the one in base?

```suggestion
```